packages/mux-player/src/styles.css

```diff
@@ -30,6 +30,13 @@ media-control-bar [role="switch"] {
   background-color: var(--media-control-background, rgba(20, 20, 30, 0.7));
 }
 
+.mxp-background {
+  height: 100%;
+  width: 100%;
+  position: absolute;
+  background-color: rgba(0, 0, 0, 0.6);
```

Comment:
design calls for 0.5, but I realized that we have text in the control bar, the time displays, and at 0.5 our contrast ratio is around 3, where-as with 0.6 we're at 5.74 which is definitely above the 4.5 WCAG requirement for text https://www.w3.org/WAI/WCAG21/Techniques/general/G18.html and the 3 req for icons https://www.w3.org/WAI/WCAG21/Techniques/general/G207.html